cpp/collections_module/algorithm/algorithm.cpp

```diff
@@ -0,0 +1,30 @@
+#include "algorithm.hpp"
+
+#include <unordered_set>
+#include "mgp.hpp"
+
+void Collections::ContainsAll(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+
+  try {
+    const auto &list1{arguments[0].ValueList()};
+    std::unordered_set<mgp::Value> set{list1.begin(), list1.end()};
+    const auto &list2{arguments[1].ValueList()};
+    std::unordered_set<mgp::Value> values{list2.begin(), list2.end()};
+
+    bool contained{true};
+    for (const auto &value : values)
+      if (set.find(value) == set.end()) {
+        contained = false;
+        break;
+      }
```

Comment:
can you use here one of the following: https://en.cppreference.com/w/cpp/container/unordered_set/operator_cmp

or https://en.cppreference.com/w/cpp/algorithm/all_any_none_of, you check if every element is in another set?

Reply:
Changed to all_of